Lca adders

m-jamieson opened the pull request:
Updated the generation mix to include MIXED power plants.
Changed the error handling in alt_generation to catch numpy warnings.
Cleaned up sheetname keywords in read_csv calls.

m-jamieson committed:
Implemented some changes to keep plants that
don't meet minimum percentfuel requirements
as mixed plants and keep them in mixes

m-jamieson committed:
Correct to account for "percentage"

m-jamieson committed:
updated to settings for 2016 electricity baseline

m-jamieson committed:
added the model_spec[region_agg] in aggregate

m-jamieson committed:
Made changes to catch numpy warnings and skip

m-jamieson committed:
cleaning up deprecated sheetname in read_csv

m-jamieson committed:
fixing missing module name

gschivley commented on electricitylci/__init__.py:
```diff
@@ -407,7 +407,8 @@ def aggregate_gen(gen_df, subregion="BA"):
     """
     import electricitylci.alt_generation as alt_gen
-
+    if subregion is None:
```
Change parameter default to None?

m-jamieson replied:
Yeah I'm fine with that.

gschivley commented:
@jump2conclusionsmatt I don't have time to test the code but it generally looks good. Did you confirm that it still works?

m-jamieson commented:
Everything should be fine. I'm actually getting the settings together to run through the code_testing spreadsheet I put up on slack.

m-jamieson commented:
I should add that the consumption mix calculations will be changed in the next business day or 2. We realized that the FERC consumption mixes should connect to balancing authorities rather than FERC regions to provide a more correct calculation of emissions. Anyhow, it may be worth holding off on the merge until that's done.

m-jamieson committed:
Changes to allow not including upstream fuels

m-jamieson committed:
Was getting errors trying to run BA with some of the
test cases - I'm sure this isn't actually needed now, but
it's working with it - so why not.

gschivley commented:
I'm going to move my EPA PR from draft to ready for review. Additional changes here will flow through once they're finalized.

m-jamieson committed:
changes to make ferc consumption pull BAs

m-jamieson committed:
flows were getting erroneously dropped due to
duplicate indeces in dataframe

m-jamieson committed:
adding testing configurations to gitignore

m-jamieson committed:
changes to support new coal inventory

m-jamieson committed:
Added a check of generation using eia923/860 data

m-jamieson committed:
new ng and coal inventories

m-jamieson committed:
Some changes for NETL

gschivley commented:
I haven't had time to look through but am accepting since it's needed for the large pull to EPA.
